Prepare 1.1.0 release

sbrandhsn opened the pull request:
### Summary

This commit prepares the repo for the 1.1.0 release. It updates the version numbers to indicate it is the final 1.1.0 release, and updates the release notes to prepare them for publishing. Once this merges this commit should be used to tag the 1.1.0 release.


### Details and comments

TODO:
- [x] Select topics for prelude draft
- [x] Finish prelude draft
- [x] Go through and copy-edit and update the release notes
- [x] Fix Sphinx Crossrefs

sbrandhsn committed:
draft 1.1.0

sbrandhsn committed:
fixes text/typos

sbrandhsn committed:
.

sbrandhsn committed:
.

mtreinish commented on releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml:
```diff
@@ -1,5 +1,5 @@
 ---
-features:
+upgrade_circuits:
```
The upgrade section is normally reserved to document changes that might require an end user to make a change in their usage when upgrading from 1.0.x -> 1.1.0. In this case the existing behavior is unchanged this is a new flag on the `power()` method that can be used, but any existing use of it wouldn't change.

```suggestion
features_circuits:
```

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,26 @@
+
+      * Improvements to the primitives interface. In particular, :class:`~.SamplerPubResult` was introduced
+        to give access to the :meth:`.SamplerPubResult.join_data` method, :class:`.DataBin` was made into a mapping class,
+        and :class:`.BitArray` was given many new utility methods.
```
I'd maybe highlight the Backend v2 primitive implementations here too.

ihincks replied:
Thanks! Yes, of course.

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,26 @@
+        that include resets and :class:`.OptimizeAnnotated` improves the processing of annotated operations.
+
+      * The default pass managers have been improved by extending them with the newly introduced transpiler
+        passes, a reorganization of existing passes and a fine-tuning of transpilation parameters.
```
I would reword this to highlight that optimization level 2 in particular has been refactored to be a better tradeoff between runtime and optimization effort to use as a default level for transpilation (and will be the default in a future release). Specifically level 2 is where we made the extensive changes and those changes in this release are really to set the stage towards making it the default.

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,26 @@
+
+      * Two synthesis steps essential for generating hardware-conforming quantum circuits have been
+        implemented in rust. This yields significant runtime speedups when decomposing unitary matrices
+        into two-qubit quantum gates using :class:`.TwoQubitBasisDecomposer` and :class:`.TwoQubitWeylDecomposition`.
```
We also moved the numerics in the `Isometry` class to rust. It's only used in controlled unitary gate synthesis though.

mtreinish replied:
I might mention that this speeds up transpile at optimization level 2 and 3 because this ends up being a large portion of the runtime in previous releases for level 3 (running it in level 2 is new for this release because of how fast it is now) when running `UnitarySynthesis`.

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,26 @@
+      * A number of new transpiler passes have been introduced to Qiskit that yield significant runtime
+        speedups while also decreasing the size of the transpiled quantum circuits in many cases. Specifically,
+        :class:`.ElidePermutations` and :class:`.StarPreRouting` have been demonstrated to have a significant
+        impact on the routing output and runtime, :class:`.RemoveFinalReset` can improve quantum circuits
```
Maybe?
```suggestion
        impact on the routing output quality and runtime, :class:`.RemoveFinalReset` can improve quantum circuits
```

mtreinish commented on releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml:
```diff
@@ -1,9 +1,9 @@
     Added a new transpiler pass :class:`.StarPreRouting` which is designed to identify star connectivity subcircuits
     and then replace them with an optimal linear routing. This is useful for certain circuits that are composed of
-    this circuit connectivity such as Berstein Vazirani and QFT. For example:
+    this circuit connectivity such as Bernstein-Vazirani and QFT. For example:
```
I wonder if we want to have an example to show how to combine this with optimziation level 2 from `generate_preset_pass_manager` to do a better full compilation with qft circuits.

mtreinish commented on qiskit/primitives/__init__.py:
```diff
@@ -464,12 +466,16 @@
     BindingsArrayLike,
     ObservableLike,
     ObservablesArrayLike,
+    Shaped,
```
@ihincks is this intended to be a public interface. I assume so since it's documented in a release note. But it'd be good to get an ack here.

ihincks replied:
ack 👍

mtreinish reviewed (commented):
Just some initial comments from a quick skim through the release notes, I'll do a more thorough pass soon.

sbrandhsn committed:
Update releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

ihincks commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,26 @@
+
+      * Improvements to the primitives interface. In particular, :class:`~.SamplerPubResult` was introduced
+        to give access to the :meth:`.SamplerPubResult.join_data` method, :class:`.DataBin` was made into a mapping class,
+        and :class:`.BitArray` was given many new utility methods.
```
```suggestion
      * New generic primitive V2 implementations were added, :class:`~.BackendEstimatorV2` and 
        :class:`~.BackendSamplerV2`, to compliment the existing full statevector based implementations.
        
      * Improvements to the primitives interface. In particular, :class:`~.SamplerPubResult` was introduced
        to give access to the :meth:`.SamplerPubResult.join_data` method, :class:`.DataBin` was made into a mapping class,
        and :class:`.BitArray` was given many new utility methods.
```

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Ian Hincks <ian.hincks@gmail.com>

sbrandhsn committed:
up

sbrandhsn committed:
up

sbrandhsn commented:
Thanks for your comments so far, I think I addressed them in the most recent commit. :-)

mtreinish commented on releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml:
```diff
@@ -1,32 +1,46 @@
+    this circuit connectivity such as Bernstein-Vazirani and QFT. For example:
 
-      .. plot:
+    .. plot:
```
```suggestion
    .. plot::
```

mtreinish commented on releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml:
```diff
@@ -1,32 +1,46 @@
 
-      .. plot:
-         :include-source:
+    .. plot:
```
```suggestion
    .. plot::
```

mtreinish commented on releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml:
```diff
@@ -1,32 +1,46 @@
+
+    Alternatively, an existing preset pass manager can be extended by:
+
+    .. plot:
```
```suggestion
    .. plot::
```

mtreinish reviewed (commented):
This should fix the plots in the star pre-routing block. Without the second colon it's being treated as a comment by sphinx.

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,35 @@
+
+      * Improvements to the primitives interface. In particular, :class:`~.SamplerPubResult` was introduced
+        to give access to the :meth:`.SamplerPubResult.join_data` method, :class:`.DataBin` was made into a mapping class,
+        and :class:`.BitArray` was given many new utility methods.
```
The other thing which would be good to highlight here is updates to platform support. The big ones are that Python 3.8 is deprecated and will no longer be supported in 1.3.0, and arm64 macOS has been promoted to tier 1 support.

sbrandhsn committed:
Update releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/star-prerouting-0998b59880c20cef.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
prelude - add platform support

ShellyGarion commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,38 @@
+---
+prelude: >
```
do you think that maybe the improvements for MCX and MCPhase gates synthesis should also appear in the prelude?
see: https://github.com/Qiskit/qiskit/pull/11993

sbrandhsn replied:
Yup, thanks for that comment - I added this to the prelude! :)

sbrandhsn committed:
prelude update

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,39 @@
+        and :class:`.BitArray` was given many new utility methods.
+
+      * Changes to platform support: python 3.8 is deprecated starting with Qiskit 1.1.0 and will no longer be supported in 1.3.0,
+        and arm64 macOS has been promoted to tier 1 support.
```
I forgot to write the release note about the changes in windows support. Can we tack this on here as an upgrade note:

```suggestion
        and arm64 macOS has been promoted to tier 1 support.
upgrade_misc:
  - |
    The minimum supported version of Windows is now Windows 10. In previous releases we did not explicitly list
    a minimum supported version of Windows and implicitly Windows 7, 8, and 8.1 may have worked (but were
    never tested). But do to Rust 1.78 dropping support for older versions of Windows Qiskit's published binaries
    on PyPI will not support older versions of Windows in this release. If you're using an older version of Windows
    you can likely still build Qiskit from source using an older rust compiler (Qiskit's minimum supported rust
    version for building from source is currently 1.70) but older versions of Windows are not a supported platform
    and are untested.
```

mtreinish commented on releasenotes/notes/1.1/added-parameter-ctrl_state-mcx-816dcd80e459a5ed.yaml:
```diff
@@ -1,10 +1,10 @@
+features_circuits:
   - |
-    Added `ctrl_state` parameter to :func:`QuantumCircuit.mcx()`.
+    Added `ctrl_state` parameter to :meth:`.QuantumCircuit.mcx`.
```
```suggestion
    Added a new ``ctrl_state`` argument to :meth:`.QuantumCircuit.mcx`.
```

mtreinish commented on releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml:
```diff
@@ -1,5 +1,5 @@
-features:
+features_circuits:
   - |
     The methods :meth:`~qiskit.circuit.QuantumCircuit.power`,
```
```suggestion
    The methods :meth:`.QuantumCircuit.power`,
```

mtreinish commented on releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml:
```diff
@@ -1,5 +1,5 @@
+features_circuits:
   - |
     The methods :meth:`~qiskit.circuit.QuantumCircuit.power`,
     :meth:`~qiskit.circuit.Gate.power`, as well as the similar methods
```
```suggestion
    :meth:`.Gate.power`, as well as the similar methods on subclasses
```

mtreinish commented on releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml:
```diff
@@ -8,10 +8,11 @@ features:
     argument ``annotated``.
```
```suggestion
    have a new argument ``annotated`` which is used to return an :class:`.AnnotatedOperation`
    object when applying a power to a gate or circuit.
```

mtreinish commented on releasenotes/notes/1.1/add-ctrl_state-mcp-parameter-b23562aa7047665a.yaml:
```diff
@@ -1,20 +1,20 @@
   - |
-    Added ``ctrl_state`` parameter to :func:`QuantumCircuit.mcp()` and
-    :class:`MCPhaseGate()`.
+    Added ``ctrl_state`` parameter to :meth:`.QuantumCircuit.mcp` and
```
```suggestion
    Added a new ``ctrl_state`` argument to :meth:`.QuantumCircuit.mcp` and
```

mtreinish commented on releasenotes/notes/1.1/add-ctrl_state-mcp-parameter-b23562aa7047665a.yaml:
```diff
@@ -1,20 +1,20 @@
 
 
-    The :func:`QuantumCircuit.mcp()` function and :class:`MCPhaseGate()` have
+    The :meth:`.QuantumCircuit.mcp` function and :class:`.MCPhaseGate` have
```
```suggestion
    The :meth:`.QuantumCircuit.mcp` method and :class:`.MCPhaseGate` class have
```

mtreinish commented on releasenotes/notes/1.1/add-elide-permutations-to-pipeline-077dad03bd55ab9c.yaml:
```diff
@@ -1,5 +1,5 @@
+features_transpiler:
   - |
     The transpiler pass :class:`~.ElidePermutations`
     runs by default with optimization level 2 and 3. Intuitively, removing
```
I'd suggest rewording the end of this release note a bit (I can't leave a suggestion because it's too far from the diff), to something like:

```
As :class:`~.OptimizeSwapBeforeMeasure` is a special case
of :class:`~.ElidePermutations`, it has been replaced by the
:class:`.ElidePermuations` pass as part of the init stage in
optimization level 3 pass managers.

mtreinish replied:
```suggestion
    runs by default in the init stage for optimization levels 2 and 3. Intuitively, removing
```

mtreinish commented on releasenotes/notes/1.1/abstract-commutation-analysis-3518129e91a33599.yaml:
```diff
@@ -1,12 +1,12 @@
+    performed on physical circuits. In addition, each operation is now checked for its ability to be
+    cached in the session commutation library. For example, this now enables computing
+    whether :class:`.AnnotatedOperation` commute. This enables transpiler passes that rely on :class:`~.CommutationChecker` internally,
     such as :class:`.CommutativeCancellation`, during earlier stages of a default transpilation pipeline
```
```suggestion
    such as :class:`.CommutativeCancellation`, to run during earlier stages of a default transpilation pipeline
```

mtreinish commented on releasenotes/notes/1.1/added-parameter-ctrl_state-mcx-816dcd80e459a5ed.yaml:
```diff
@@ -1,10 +1,10 @@
 
 
-    The :func:`QuantumCircuit.mcx()` function in the quantum circuit library has
+    The :meth:`.QuantumCircuit.mcx` function in the quantum circuit library has
```
```suggestion
    The :meth:`.QuantumCircuit.mcx` method in the quantum circuit library has
```

mtreinish commented on releasenotes/notes/1.1/classical-store-e64ee1286219a862.yaml:
```diff
@@ -44,8 +44,8 @@ features_circuits:
-    in a circuit using :meth:`.QuantumCircuit.add_var` and :meth:`~.QuantumCircuit.add_input`, or
-    at a lower level using :meth:`.Var.new`.
+    in a circuit using :meth:`.QuantumCircuit.add_var` and :meth:`.QuantumCircuit.add_input`, or
+    at a lower level using :func:`~.qiskit.circuit.classical.expr.Var.new`.
```
I think this was correct before. The cross ref is broken looking at the artifact locally.
```suggestion
    at a lower level using :func:`.Var.new`.
```

mtreinish commented on releasenotes/notes/1.1/fixes_10852-e197344c5f44b4f1.yaml:
```diff
@@ -1,5 +1,5 @@
-    The :class:`.BasicSimulator` python-based simulator included in :mod:`qiskit.providers.basic_provider`
-    now includes all the standard gates (:mod:`qiskit.circuit.library .standard_gates`) up to 3 qubits.
+    The :class:`.BasicSimulator` python-based simulator included in :mod:`~.qiskit.providers.basic_provider`
+    now includes all the standard gates (:mod:`~.qiskit.circuit.library.standard_gates`) up to 3 qubits.
```
```suggestion
    now supports running all the standard gates up to 3 qubits defined in :mod:`qiskit.circuit.library`.
```

mtreinish commented on releasenotes/notes/1.1/optimize-annotated-conjugate-reduction-656438d3642f27dc.yaml:
```diff
@@ -1,4 +1,4 @@
+features_transpiler:
   - |
     Added a new reduction to the :class:`.OptimizeAnnotated` transpiler pass.
     This reduction looks for annotated operations (objects of type :class:`.AnnotatedOperation`
```
I wonder if we want to use math mode instead of monospace mode for the letters like `B` below (I can't leave inline suggestions on that).

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,39 @@
+
+      * Improvements to the primitives interface. In particular, :class:`~.SamplerPubResult` was introduced
+        to give access to the :meth:`.SamplerPubResult.join_data` method, :class:`.DataBin` was made into a mapping class,
+        and :class:`.BitArray` was given many new utility methods.
```
I might drop this bullet point the highlights list is getting pretty long and this seems less important and more in the weeds than the new backend primitive v2s.

```suggestion
```

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,39 @@
+        level 2 and 3 that incurred a large runtime overhead for these pass managers historically.
+        Furthermore, the numeric methods used in :class:`.Isometry` have been moved to rust, enabling large
+        runtime speed-ups in particular for controlled unitary gate synthesis. The decomposition for multi-controlled
+        not and phase gates has been improved by more than two orders of magnitude.
```
Just a few suggestions on wording, you don't have to take this verbatim but I think this was a bit akward to read. At the end of the day what er

```suggestion
      * The default two qubit synthesis methods that are used internally by the transpiler in the
        :class:`.UnitarySynthesis` pass have been re-implemented in Rust. This yields significant
        runtime speedups when decomposing two qubit unitary matrices. This corresponds
        to a large improvement in the runtime for transpilation in level 3 where running
        :class:`.UnitarySynthesis` for two qubit synthesis incurred a large runtime overhead
        historically. This release also starts running :class:`.UnitarySynthesis` as part of the
        optimization stage in optimization level 2 because of these runtime performance
        improvements.
       
        Additionally, the numeric methods used in :class:`.Isometry` have been moved to rust,
        enabling large runtime speed-ups in particular for controlled unitary gate synthesis.
        The decomposition for multi-controlled :class:.XGate` and :class:`.PhaseGate`  has been
        improved resulting in a reduction in the number of gates used in the synthesis by more
        than two orders of magnitude.
```

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,39 @@
+        speedups while also decreasing the size of the transpiled quantum circuits in many cases. Specifically,
+        :class:`.ElidePermutations` and :class:`.StarPreRouting` have been demonstrated to have a significant
+        impact on the routing output quality and runtime, :class:`.RemoveFinalReset` can improve quantum circuits
+        that include resets and :class:`.OptimizeAnnotated` improves the processing of annotated operations.
```
`OptimizeAnnotated` isn't new, but it has been improved in this release

sbrandhsn replied:
I removed that pass from the prelude. :-)

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

mtreinish commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,39 @@
+      * The default pass managers have been improved by extending them with the newly introduced transpiler
+        passes, a reorganization of existing passes and a fine-tuning of transpilation parameters. In particular,
+        the optimization-level-2 preset pass manager has been refactored to have a better tradeoff between
+        the runtime and optimization effort in order to serve as a default pass manager in future releases.
```
I would just highlight the level 2 changes here mostly. Using elide permutations and commutative cancellation in init on level 3 are fairly minor compared to the reworking of level 2.

```suggestion
      * The default pass managers have been improved by extending them with the newly introduced transpiler
        passes. In particular, the optimization level 2 preset pass manager from
        :class:`.generate_preset_pass_manager` and used internally by :func:.transpile` has been refactored to
        have a better tradeoff between runtime and optimization effort in order to serve as a default pass
        manager in future releases. While this release doesn't change the default to use level 2 it is a better
        choice than using level 1 or 3.
```

mtreinish reviewed (commented):
I did a pass through all the notes (except the fix notes) that were in the PR diff to start. I still want to go through the rest of the notes quickly to see if anything else stands out to me.

sbrandhsn committed:
Update releasenotes/notes/1.1/added-parameter-ctrl_state-mcx-816dcd80e459a5ed.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/add-ctrl_state-mcp-parameter-b23562aa7047665a.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Apply suggestions from code review

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/abstract-commutation-analysis-3518129e91a33599.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
comments from code review

sbrandhsn commented:
Thanks for the review so far, I included the requested changes. :-)

ihincks commented on qiskit/primitives/containers/shape.py:
```diff
@@ -60,14 +60,17 @@ def __repr__(self):
     @property
     def size(self):
+        """size (int): The product of the entries in the shape attribute."""
         return int(np.prod(self._shape, dtype=int))
```
This inherits from `Shaped` and can safely be typed as such.

```suggestion
    @property
    def shape(self) -> tuple[int, ...]:
        """The array shape of this object."""
        return self._shape

    @property
    def ndim(self) -> int:
        """The length of the shape attribute."""
        return len(self._shape)

    @property
    def size(self) -> int:
        """The product of the entries in the shape attribute."""
        return int(np.prod(self._shape, dtype=int))
```

Because of the inheritance, do we actually need docstrings?

```suggestion
    @property
    def shape(self) -> tuple[int, ...]:
        return self._shape

    @property
    def ndim(self) -> int:
        return len(self._shape)

    @property
    def size(self) -> int:
        return int(np.prod(self._shape, dtype=int))
```

sbrandhsn committed:
Update qiskit/primitives/containers/shape.py

Co-authored-by: Ian Hincks <ian.hincks@gmail.com>

sbrandhsn committed:
Update shape.py

1ucian0 commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,52 @@
+        :class:`~.BackendSamplerV2`, to compliment the existing full statevector based implementations.
+
+
+      * Changes to platform support: python 3.8 is deprecated starting with Qiskit 1.1.0 and will no longer be supported in 1.3.0,
```
Previous release notes capitalize Python

```suggestion
      * Changes to platform support: Python 3.8 is deprecated starting with Qiskit 1.1.0 and will no longer be supported in 1.3.0,
```

1ucian0 commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,52 @@
+        starts running :class:`.UnitarySynthesis` as part of the optimization stage in optimization level
+        2 because of these runtime performance improvements.
+
+        Additionally, the numeric methods used in :class:`.Isometry` have been moved to rust,
```
some other instances of the name are capitalized.

```suggestion
        Additionally, the numeric methods used in :class:`.Isometry` have been moved to Rust,
```

1ucian0 commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,52 @@
+    a minimum supported version of Windows and implicitly Windows 7, 8, and 8.1 may have worked (but were
+    never tested). But do to Rust 1.78 dropping support for older versions of Windows Qiskit's published binaries
+    on PyPI will not support older versions of Windows in this release. If you're using an older version of Windows
+    you can likely still build Qiskit from source using an older rust compiler (Qiskit's minimum supported rust
```
```suggestion
    you can likely still build Qiskit from source using an older Rust compiler (Qiskit's minimum supported Rust
```

sbrandhsn committed:
Apply suggestions from code review

Co-authored-by: Luciano Bello <bel@zurich.ibm.com>

1ucian0 commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,52 @@
+        :class:`.generate_preset_pass_manager` and used internally by :func:`.transpile` has been refactored to
+        have a better tradeoff between runtime and optimization effort in order to serve as a default pass
+        manager in future releases. While this release doesn't change the default to use level 2 it is a better
+        choice than using level 1 or 3.
```
I guess not always.. right?

mtreinish replied:
```suggestion
        manager in future releases. While this release doesn't change the default to use level 2 it is typically a better
        choice than using level 1 or 3.
```

1ucian0 commented on releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml:
```diff
@@ -0,0 +1,52 @@
+  - |
+    The minimum supported version of Windows is now Windows 10. In previous releases we did not explicitly list
+    a minimum supported version of Windows and implicitly Windows 7, 8, and 8.1 may have worked (but were
+    never tested). But do to Rust 1.78 dropping support for older versions of Windows Qiskit's published binaries
```
?
```suggestion
    never tested). But due to Rust 1.78 dropping support for older versions of Windows Qiskit's published binaries
```

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

sbrandhsn committed:
Update releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

Co-authored-by: Luciano Bello <bel@zurich.ibm.com>

sbrandhsn committed:
.

sbrandhsn committed:
.

mtreinish committed:
Move and update new fix release notes

mtreinish commented:
I had to do a manual rebase and force push to update the branch here because reno has a hard time tracking things across a merge commit in the PR. The only real change from the prior version is https://github.com/Qiskit/qiskit/pull/12371/commits/2e749fbe74fe92c4be87ec7c4abe9c6fe0cfb1be

mtreinish committed:
Small tweaks

mtreinish committed:
Remove fix note for private API

jakelishman committed:
Add release notes for manual `Var` and `Store` (#12421)

* Add release notes for manual `Var` and `Store`

This adds the release notes and updates some small portions of
documentation that were previously missed surrounding the new manual
`Var` storage locations.

This includes documenting all new keyword arguments to methods, upgrade
instructions for providers, and adding the `Var.new` method to the
documentation, which was previously erroneously omitted.

* Fix Sphinx typo

* Fix another Sphinx typo

* Move QPY version bump to upgrade

* Unify base release note

* Reword providers upgrade note

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>

---------

Co-authored-by: Matthew Treinish <mtreinish@kortar.org>
(cherry picked from commit 554e661ee62ba6db1cfcbef0fabd98a1659a1641)

mtreinish reviewed (approved):
This LGTM, there might be some other small things in the release notes but we can fix this after the fact fairly easily. Thanks for the hard work taking care of this and shepherding this release @sbrandhsn!
